ui/userguide-view/ui/CardBody.jsx

```diff
@@ -0,0 +1,207 @@
+import React from 'react';
+
+import ShortcutTable from '../../shared/ShortcutTable';
+
+class CardBody extends React.Component {
+  componentWillMount() {
+    // Used for dynamic startdate, enddate param examples
+    const d = new Date();
+    const y = d.getFullYear();
+    const m = d.getMonth() + 1;
+    const sd = d.getDate() - 2;
+    const ed = d.getDate();
+
+    this.endDate = `${y}-${m}-${ed}`;
+    this.startDate = `${y}-${m}-${sd}`;
+  }
+
+  render() {
+    const possiblesNotation = [
```

Comment:
This would be better defined outside the class, rather than in ``render()``  Basically just a file-level constant that's used only by this class.